pkg/scheduler/framework/plugins/nodevolumelimits/OWNERS

```diff
@@ -0,0 +1,10 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+  - sig-storage-approvers
+  - cofyc
```

Comment:
We added cofyc specifically to volumebinding as he did significant contributions there.

But for the rest of the plugins, I'll defer to @msau42 for endorsement.

Reply:
or to add to sig-storage aliases :)